src/ImageSharp.Drawing/Processing/Drawing/Brushes/GradientBrushes/EllipticGradientBrush.cs

```diff
@@ -0,0 +1,167 @@
+﻿using System;
+
+using SixLabors.ImageSharp.PixelFormats;
+using SixLabors.Primitives;
+
+namespace SixLabors.ImageSharp.Processing.Drawing.Brushes.GradientBrushes
+{
+    /// <summary>
+    /// Gradient Brush with elliptic shape.
+    /// The ellipse is defined by a center point,
+    /// a point on the longest extension of the ellipse and
+    /// the ratio between longest and shortest extension.
+    /// </summary>
+    /// <typeparam name="TPixel">The Pixel format that is used.</typeparam>
+    public class EllipticGradientBrush<TPixel> : AbstractGradientBrush<TPixel>
```

Comment:
thanks, commit follows.

Didn't know that but read something about the possible optimizations for sealed classes. Learned something new again (I know now that it is as you said, and some of the why; although I'm not entirely clear why those optimizations should not be possible for non-sealed classes).

In general I don't like sealed because you never know what someone else might want to do with your code, and I don't like to forbid other developers more than I have to; but here is a reason, so I'm fine with that.